zuul.d/ansible-security-jobs.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- job:
+    name: ansible-security-integration-ibm-qradar
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    parent: ansible-cloud-appliance-base
+    pre-run:
+      - playbooks/ansible-test-base/pre.yaml
+      - playbooks/ansible-test-network-integration-base/pre.yaml
```

Comment:
We probably don't need this line since we don't start any appliance VM. You can remove it.